packages/flutter_tools/test/commands/doctor_test.dart

```diff
@@ -36,4 +36,7 @@ class IntelliJValidatorTestTarget extends IntelliJValidator {
 
   @override
   String get version => 'test.test.test';
+
+  @override
+  String get buildNumber => 'IU-173.3942.27';
```

Comment:
Does this mean that the test will now issue a real HTTP request to JetBrains?  We should mock that out, as we're striving to make all our tests hermetic (thus removing potential flakiness)